Documentation/Index.rst

```diff
@@ -1,69 +1,55 @@
+..  include:: /Includes.rst.txt
 
-=============
-render-guides
-=============
+.. _start:
 
-Mono-repo setup
-===============
+=========
+<project>
+=========
 
-This repository is following a mono-repo setup. This means all code and
-configuration to render documentation is in this repository. This includes
-scripts to build the documentation, the configuration for the CI/CD pipeline.
+:Package name:
+    render-guides
 
-To ensure the mono-repo setup works, and also works in separate repositories,
-we are using a tool called monorepo-buider_. This tool will help us to keep the
-dependencies over packages in sync.
+:Version:
+    |release|
 
-If you add a new dependency to a package, you can run
+:Language:
+    en
 
-::
-    composer run monorepo:merge
+:Author:
+    TYPO3 contributors
```

Comment:
Maybe "TYPO3 documentation contributors"? What do you think, @linawolf ?

Reply:
I think we can leave it like that